scene/2d/gpu_particles_2d.cpp

```diff
@@ -319,6 +319,10 @@ PackedStringArray GPUParticles2D::get_configuration_warnings() const {
 		warnings.push_back(RTR("Particle trails are only available when using the Forward+ or Mobile rendering backends."));
 	}
 
+	if (sub_emitter != NodePath() && OS::get_singleton()->get_current_rendering_method() == "gl_compatibility") {
+		warnings.push_back(RTR("Particle sub emitters are only available when using the Forward+ or Mobile rendering backends."));
```

Comment:
I didn't notice this one before, apologies:

```suggestion
		warnings.push_back(RTR("Particle sub-emitters are only available when using the Forward+ or Mobile rendering backends."));
```